CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 1.2.0
+
+## Features and improvements
+* Add `*-sort-key` layout property for circle, fill, and line layers, to sort the z-index of features within a single layer([#8467](https://github.com/mapbox/mapbox-gl-js/pull/8467))
+* Add ability to instantiate maps with specific access tokens ([#8364](https://github.com/mapbox/mapbox-gl-js/pull/8364))
+* Accommodate `prefers-reduced-motion` settings in browser ([#8494](https://github.com/mapbox/mapbox-gl-js/pull/8494))
+* Add Map `visualizePitch` option that tilts the compass as the map pitches ([#8208](https://github.com/mapbox/mapbox-gl-js/issues/8208), fixed by [#8296](https://github.com/mapbox/mapbox-gl-js/pull/8296)) (h/t [pakastin](https://github.com/pakastin))
+* Make source options take precedence over TileJSON ([#8232](https://github.com/mapbox/mapbox-gl-js/pull/8232)) (h/t [jingsam](https://github.com/jingsam))
+* Make requirements for text offset properties more precise ([#8418](https://github.com/mapbox/mapbox-gl-js/pull/8418))
+* Expose `convertFilter` API in the style specification ([#8493](https://github.com/mapbox/mapbox-gl-js/pull/8493)
+
+## Bug fixes
+* BREAKING: Fix changes to `text-variable-anchor`, such that previous anchor positions would take precedence only if they are present in the updated array (considered a bug fix, but is technically a breaking change from previous behavior) ([#8473](https://github.com/mapbox/mapbox-gl-js/pull/8473))
```

Comment:
Hmmm, was this change made based on feedback from @mourner or @asheemmamoowala? If so, LGTM. 

If not: specifying a breaking change like this is confusing to me as a user. I would prefer the following:
- Remove "BREAKING:" from the changelog and leave it in the bug fix section with the parenthetical note as-is, OR
- Flag as a breaking change in its own section like we would usually do (see https://github.com/mapbox/mapbox-gl-js/releases/tag/v1.0.0 for an example) and note that it also fixes a bug

I prefer option 1 -- I don't think that the impact of this change is large or will break customers' maps in unexpected ways.